helm/managed/logging/elastic/values.yaml

```diff
@@ -21,6 +28,35 @@ elasticsearch:
       # Add default labels for the volumeClaimTemplate fo the StatefulSet
       #enabled: false
     #annotations: {}
+  extraEnvs:
+    - name: WAIT_ES_READY_IN_SEC
+      value: "100"
+    #- name: STATUS_COLOR
+    #  value: "yellow"
+  lifecycle:
+    postStart:
+      exec:
+        command:
+          - bash
+          - -c
+          - |
+            PIPELINE_NAME=hpccpipeline
+            ES_URL=http://localhost:9200
+            ES_CLUSTER_HEALTH_PATH=_cluster/health
+            ES_CLUSTER_HEALTH_PARAMS=_cluster/health
+            PIPELINE_REQ_PATH=_ingest/pipeline
+            [ -z ${WAIT_ES_READY_IN_SEC} ] && WAIT_ES_READY_IN_SEC=100
+            END_TIME=$(expr $(date +%s) \+ ${WAIT_ES_READY_IN_SEC})
+            ##START_FILE=/tmp/.es_start_file
+            ##ls $START_FILE
+            while [[ "$(curl -s -o /dev/null -w '%{http_code}\n' $ES_URL)" != "200" ]]; do
+              echo "\nES not ready..."
+              [ $(date +%s) -ge ${END_TIME} ] && break
```

Comment:
what's the behavior if this times out?

Reply:
It will continue for current setup. I am not sure your plan if it times out

Reply:
For this change to be valuable, we have to ensure the pipeline is created, otherwise we should abort. How can we control that in a clean manner?